cmd/kube-apiserver/app/server.go

```diff
@@ -277,6 +279,10 @@ func Run(s *options.ServerRunOptions) error {
 	genericConfig.EnableOpenAPISupport = true
 	genericConfig.EnableMetrics = true
 	genericConfig.OpenAPIConfig.SecurityDefinitions = securityDefinitions
+	genericConfig.LongRunningFunc = filters.BasicLongRunningRequestCheck(
```

Comment:
Do we care about thsi being configurable at all any more? Could we just bake it into genericapiserver and anyone who wants something different can take control of the handler chain?